pkg/kubernetes/scripts/kube-client-cockpit.js

```diff
@@ -38,7 +38,13 @@
         try {
             obj = JSON.parse(response.data);
         } catch(e) {
-            return;
+            // Some kubernetes versions message up json reponses
```

Comment:
This is supposed to be "mess up", I figure. This smells like a regression in 3.7rc. However, `updateMessage()` is called for failed requests to opportunistically pry out some useful error message, so as long as we are  sure that the actual contents is a sensible error message (as opposed to e. g. a huge configuration or log text dumps), it makes sense to accept plaintext errors as well.

Reply:
It's the json message forced to a string, this has the downside of repeating the error text twice. But this is better than not showing any error message which is the only other option.

It's a kubernetes bug and it's fixed in 1.8. I'm trying but it seems unlikely that they'll agree to backport.